user/reference/precise.md

```diff
@@ -7,7 +7,8 @@ redirect_from:
   - /user/workers/standard-infrastructure/
 ---
 
-> Precise is EOL by Canonical, try updating to a newer image and see our [Precise to Trusty Migration Guide](/user/precise-to-trusty-migration-guide) and [Trusty to Xenial Migration Guide](/user/trusty-to-xenial-migration-guide).
+> Precise is EOL by Canonical, try updating to a newer image. It is considered as deprecated build envrionment in Travis CI and not supportted with any updates.
```

Comment:
Precise is EOL by Canonical, try updating to a newer image. It is considered as deprecated build environment in Travis CI and is not supported with any updates.